example/index.ts

```diff
@@ -46,14 +50,14 @@ async function runFeature() {
   return runtime.start()
 }
 
-function appendToOutput(data) {
+function appendToOutput(data): void {
```

Comment:
Can a type be added for `data`

Reply:
For this initial PR, I'd rather not worry about typing everything and just getting things passing and then make incremental improvements afterwards